src/assets/styles.css

```
@@ -75,3 +80,239 @@ select{
+    font-feature-settings: 'tnum' on, 'lnum' on;
+    
+    color: #6B7280;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.ActionBtns{
+    
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 25px;
+
+    width: 598px;
+    height: 53px;
+
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+}
+
+.CTAbtn {    
+
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 10px 40px;
+    gap: 10px;
+    width: 272px;
+    height: 53px;
+    border-radius: 30px;
+    background: #2563EB;
+    border: 3px solid #2563EB;
+    transition: 400ms;
+
+    /*Text*/
+    color: #FFFFFF;
+    font-size: 24px;
+       
+        /* Inside auto layout */
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+}
+
+.LearnMorebtn{
+   
+    box-sizing: border-box; 
+    justify-content: center;
+    align-items: center;
+    padding: 10px 40px;
+    gap: 10px;
+    width: 272px;
+    height: 53px;
+    border-radius: 30px;
+    background: none;
+
+    /*Text*/
+    font-size:var(--Normal-subHeaders-size);
+    color: #6B7280;
+
+    /* Inside auto layout */
+        
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    border: 3px solid #6B7280;
+    
+}
+
+.CTAbtn:hover {
+    background: #0c3693;
+    border: #0c3693;
+    cursor: pointer;
+}
+
+.LearnMorebtn:hover{
+    border-color: rgb(30, 23, 23);
+    color: rgb(30, 23, 23);
+    cursor: pointer;
+}
+
+.features{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between 30px;
+    align-items: center;
+    gap: 30px;
+    top: 950px;
+}
+
+.feature2{
+   
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: start;
+    gap: 50px;
+    padding: 0vh 15vh 0vh 15vh;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+.feature1, .feature3{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between ;
+    align-items: start;
+    gap: 20px;
+    padding: 0vh 15vh 0vh 15vh;
+    flex: none;
+    order: 0;
+    align-self: stretch;
+    flex-grow: 0;
+}
+
+
+
+#feat1-img #feat3-img{ 
+ 
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+#feat2-img{
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+#description{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    
+    padding: 12vh 0vh 12vh 10vh;
+    gap: 20px;
+    
+    /* Inside auto layout */
+    
+    flex: none;
+    order: 0;
+    flex-grow: 1;
+}
+
+#header{
+    color: var(--primary-color);
+    font-size: var(--Normal-big-headers-size);
+    font-weight: 700;
+    /* display: flex;
+    align-items: flex-start; */
+    /* padding: 0vh 10vh 10vh;
+    margin-top: 10vh; */
+}
+
+#info{
+    width: 614px;
+height: 54px;
+font-style: normal;
+font-weight: 400;
+font-size: var(--Normal-subHeaders-size);
+line-height: 130%;
+
+}
```

```suggestion
#info{
    width: 614px;
    height: 54px;
    font-style: normal;
    font-weight: 400;
    font-size: var(--Normal-subHeaders-size);
    line-height: 130%;
}
```
Και αυτό να γίνει class.